client/src/components/TournamentRow.tsx

```diff
@@ -14,18 +14,14 @@ export enum TournamentStatus {
   Finished,
 }
 
-export class TournamentRow extends React.Component<TournamentRowProps, {}> {
-  constructor(props: TournamentRowProps) {
-    super(props)
-    console.log(TournamentStatus[this.props.tournament.status])
-  }
+export function TournamentRow(props: TournamentRowProps): JSX.Element {
+  const [tournament] = useState(props.tournament)
```

Comment:
I think we don't need to keep the `tournament` in the state. Could we instead read it directly from the props? Or do we expect to change the `tournament` from inside this component?